lighthouse-core/test/computed/metrics/responsiveness-test.js

```diff
@@ -140,14 +166,212 @@ describe('Metric: Responsiveness', () => {
       .rejects.toThrow();
   });
 
+  it('throws if there are Responsiveness events but no EventTiming events', async () => {
+    const interactionEvents = [{
+      ts: 500,
+      maxDuration: 200,
+    }];
+    const trace = makeTrace(interactionEvents);
+    trace.traceEvents = trace.traceEvents.filter(e => e.name !== 'EventTiming');
+
+    const metricInputData = {
+      trace,
+      settings: {throttlingMethod: 'provided'},
+    };
+    expect(Responsiveness.request(metricInputData, {computedCache: new Map()}))
```

Comment:
```suggestion
    await expect(Responsiveness.request(metricInputData, {computedCache: new Map()}))
```
+ other calls like this in this file.

Reply:
good catch, thanks!